cwl/workflow/scripts/gatk-create-interval-lists.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
```

Comment:
there are two copies of this script, one in docker and one here - there should only be one.